app/src/main/java/com/github/se/cyrcle/ui/map/MapScreen.kt

```diff
@@ -187,27 +178,49 @@ fun MapScreen(
       }
 }
 
+fun inBounds(
```

Comment:
I think there is a missing documentation here.

Reply:
fixed